lib/sacastats_web.ex

```diff
@@ -78,6 +78,36 @@ defmodule SacaStatsWeb do
     end
   end
 
+  def generate_page_title(endpoint) do
+    site_name = "Saca Stats"
+    separator = " - "
+    postfix = separator <> site_name
+    endpoint_arr = String.split(endpoint, "/", trim: true)
+
+    case length(endpoint_arr) do
+      0 -> site_name
+      _ -> generate_subpage_title(endpoint_arr, separator) <> postfix
+    end
```

Comment:
For binary `case`s (only 2 outcomes), it can look nicer to use if/else
```suggestion
    if length(endpoint_arr) == 0 do
      site_name
    else
      generate_subpage_title(endpoint_arr, separator) <> postfix
    end
```

Reply:
I originally was going to have 3, so this is a good call.